templates/01_block-scoping.test.js

```diff
@@ -92,5 +94,20 @@ test('means that we can start using block statements', () => {
 
 //////// EXTRA CREDIT ////////
 
+test.skip('means that we can declare constant with the same name in block statement', () => {
+  // WORKSHOP_START
+  // Declare 'd' using 'const', setting the value to 5
+  // In a block statement declare 'd' using 'const', setting the value to 10
+  // WORKSHOP_END
+  // FINAL_START
+  const d = 5
+  // BLOCK STATEMENT
+  {
+    const d = 10
+  }
+  // FINAL_END
+  expect(d).toBe(5)
```

Comment:
Could we do a `ENTER YOUR GUESS HERE` inline comment so they have something to do here :+1: